hummingbot/strategy/pure_market_making/constant_size_sizing_delegate.pyx

```diff
@@ -36,20 +41,27 @@ cdef class ConstantSizeSizingDelegate(OrderSizingDelegate):
                                           object pricing_proposal):
         cdef:
             MarketBase market = market_info.market
+            object buy_fees
             double base_asset_balance = market.c_get_balance(market_info.base_currency)
             double quote_asset_balance = market.c_get_balance(market_info.quote_currency)
             double bid_order_size = self._order_size
             double ask_order_size = self._order_size
             bint has_active_bid = False
             bint has_active_ask = False
 
+        buy_fees = market.c_get_fee(market_info.base_currency, market_info.quote_currency,
+                                    OrderType.MARKET, TradeType.BUY,
+                                    bid_order_size, pricing_proposal.buy_order_prices[0])
+
         if market.name == "binance":
-            bid_order_size = market.c_quantize_order_amount(market_info.symbol, self.order_size, pricing_proposal.buy_order_prices[0])
+            bid_order_size = market.c_quantize_order_amount(market_info.symbol, (self.order_size * (1+float(buy_fees.percent))), pricing_proposal.buy_order_prices[0])
```

Comment:
This seems to adjust the order size for fees - I thought we are trying to not adjust the order size

Reply:
Removed Binance order adjustment as discussed